modules/hdf5_handler/USAGE

```diff
@@ -27,8 +19,10 @@ server in the following ways:
   "joinNew", or "joinExisting" aggregations.
 
  Writing an NcML file is as easy as writing an HTML file. Please refer to 
- the NcML example page [1] at the HDF-EOS Tools and Information Center website.
+ the NcML example page [3] and [4] at the HDF-EOS Tools and Information Center website.
 
 References
-
-[1] http://hdfeos.org/examples/ncml.php
+[1] https://github.com/OPENDAP/hyrax_guide/blob/master/handlers/BES_Modules_The_HDF5_Handler.adoc
```

Comment:
Can you add the link to the online Hyrax Users' Guide (https://opendap.github.io/hyrax_guide/Master_Hyrax_Guide.html#_hyrax_handlers) as well. This is version is built automatically when you push changes to the adoc file in github, so it's always up to date. It's frustrating, but I don't know how to link directly to the HDF5 info, but this is the documentation that is linked to from the NGAP landing page/

Reply:
OK, I add it and I will also add this link to hdfeos.org